src/content/basics/nodepools/index.md

```diff
@@ -189,6 +163,17 @@ In case there are workloads not assigned to any node pools, the autoscaler may p
 
 ## Limitations
 
+- Clusters without worker nodes (= without node pools) cannot be considered fully functional. In order to have all
+required components scheduled, worker nodes are required. For that reason, we deactivate any monitoring and alerts for
+these clusters and don't provide any proactive support.
+
+- We also do not monitor node pools with less than three worker nodes and do not provide any proactive support for those.
+
+- When creating a new node pool, the master node of the cluster is re-created. This causes a downtime of the Kubernetes
+API of a couple of minutes.
+
+### AWS specific
+
 - A node pool can have a maximum of 250 worker nodes. The architectural reason for this is that each node pool gets a `/24` IPv4 subnet assigned. However 5 IP addresses per availability zone used are not usable for worker nodes, as they are reserved for other AWS resources. Hence the limit depends on the number of availability zones used by a node pool. If the pool uses two zones, it's 245. With three zones, the limit is 240, and with four zones, it's 235.
```

Comment:
This is not true on Azure, because we have the 100 limitation at the VMSS level. Let me rephrase that.

Reply:
OK this was the AWS section, sorry